extensions/2.0/Vendor/EXT_structural_metadata/README.md

```diff
@@ -11,40 +13,522 @@
+> 
+> ![Property Mapping](figures/property-mapping.png)
+> 
+> _Top-level extension object:_
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_structural_metadata": {
+>       "schema": {  
+>         "classes": {
+>           "tree": {
+>             "name": "movement",
+>             "description": "The movement of points in a point cloud",
+>             "properties": {
+>               "direction": {
+>                 "description": "The movement direction, as a normalized 3D vector",
+>                 "type": "VEC3",
+>                 "componentType": "FLOAT32",
+>                 "required": true
+>               },
+>               "magnitude": {
+>                 "description": "The magnitude of the movement",
+>                 "type": "SCALAR",
+>                 "componentType": "FLOAT32",
+>                 "required": true
+>               }
+>             }
+>           }
+>         }
+>       },
+>       "propertyMappings": [{
+>         "class": "movement",
+>         "properties": {
+>           "direction": {
+>             "attribute": "_DIRECTION",
+>           },
+>           "magnitude": {
+>             "attribute": "_MAGNITUDE",
+>           }
+>         }
+>       }]
+>     }
+>   }
+> }
+> ```
+> _Primitive_
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "mode:": 0, 
+>       "attributes": {
+>         "POSITION": 0,
+>         "_DIRECTION": 1,
+>         "_MAGNITUDE": 2,
+>       },
+>       "extensions": {
+>         "EXT_structural_metadata": {
+>           "propertyMappings": [0]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+
+### Property Textures
+
+*Defined in [propertyTexture.schema.json](./schema/propertyTexture.schema.json).*
+
+Property textures use texture channels to store property values conforming to a particular class (identified by ID `class`), with those values accessed directly by texture coordinates. Property textures are especially useful when texture mapping high frequency data to less detailed 3D surfaces. Unlike textures used in glTF materials, property textures are not necessarily visible in a rendered scene. Like property tables, property textures are implementations of the [3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/main/specification/Metadata).
+
+Property textures are defined as entries in the `propertyTextures` array of the root-level `EXT_structural_metadata` extension, and may be referenced by extensions on primitive objects. 
+
+A property texture may provide channels for only a subset of the properties of its class, but class properties marked `required: true` must not be omitted.
+
+Several constraints are imposed to ensure compatibility of texture storage with various property types:
+
+* Variable-length arrays are not supported.
+* Data type and bit depth of the image must be compatible with the property type.
```

Comment:
This note and the note below about floating-point KTX2 might be obsolete now that properties values can be packed in multiple chanels.